saskatoon/member/models.py

```diff
@@ -289,8 +289,8 @@ class Organization(Actor):
     street = models.CharField(
         verbose_name=_("Street"),
         max_length=50,
-        null=True,
-        blank=True
+#         null=True,
+#         blank=True
```

Comment:
this will cause issues when migrating, if you remove null=True and blank=True, you need to provide a default

Reply:
Yes yes your are right sir it's true

Reply:
ok no worries. please don't use "sir" to refer to me. thanks!

Reply:
do you mind if we close this pull request?

Reply:
> do you mind if we close this pull request?

no @2ynn  you can close  thanks